startup_test.go

```diff
@@ -125,6 +214,38 @@ func changeSettings(uuid, token string) error {
 	return nil
 }
 
+func despawnInstance(uuid, token string) error {
+	req, err := http.NewRequest("POST", "http://127.0.0.1:8087/api/v1/bot/i/"+uuid+"/kill", nil)
```

Comment:
a lot of similar stuff which happens in despawnInstance and spawnInstance etc. We should probably create an ApiClient wrapper.